app/web/src/newhotness/logic_composables/change_set.ts

```diff
@@ -71,23 +70,16 @@ export const useChangeSets = (
 
 export const useApplyChangeSet = (ctx: Context) => {
   const api = useApi(ctx);
-  const status = useStatus();
 
   const applyInFlight = computed(() => api.inFlight.value);
+ 
   const performApply = async () => {
     const call = api.endpoint(routes.ApplyChangeSet);
     const { req } = await call.post({});
     return { success: api.ok(req) };
   };
-  const disallowApply = computed(
-    () =>
-      (ctx.changeSet.value?.status !== ChangeSetStatus.Open &&
-        ctx.changeSet.value?.status !== ChangeSetStatus.NeedsApproval) ||
-      ctx.onHead.value ||
-      status.value === "syncing",
-  );
```

Comment:
Too messy as a composable and only two spots need it now. They both have different logic too.